### ISSUE TITLE
NTR: MRI: compression of eight cranial nerve nerve by blood vessel (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8896]
New term request MRI: compression of eight cranial nerve nerve by blood vessel (jadhav_091115201135): NVCC; neurovascular cross-compression; vestibular nerve; vestibulocochlear; auditory; acoustic; 8th  UMLS: 0521670

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000845
- label: Vascular compression of the eighth cranial nerve
- parents: [Abnormality of the vestibulocochlear nerve (HP:0009591)](https://hpo.jax.org/app/browse/term/HP:0009591)
- pmid: PMID:9452229
- def: A condition in which blood vessels adjacent to the eighth cranial nerve (vestibulocochlear nerve) such as the anterior inferior cerebellar artery abnormally contact or compress the vestibulocochlear nerve as it exits the brainstem or travels within the cerebellopontine angle.
- comment: CN VIII consists of 3 components: the cochlear nerve, the superior vestibular nerve, and the inferior vestibular nerve. If the NVC syndrome of CN VIII exists, symptoms specifically reflecting the nerve component affected should be manifested. If compression by a vascular loop is limited to the cochlear nerve alone, tinnitus or a hearing disturbance (or both) should occur, whereas if the compression is limited to the vestibular nerve, vertigo should occur, and if the cochlear nerve and the vestibular nerve are compressed, the patient should experience tinnitus and vertigo at the same time.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8896
- synonyms: Neurovascular compression syndrome of the vestibulocochlear nerve(NONE)|Vestibulocochlear nerve compression syndrome(NONE)
